src/amo/components/SearchResult/index.js

```diff
@@ -132,7 +125,15 @@ export class SearchResultBase extends React.Component<InternalProps> {
     }
 
     return (
-      <div className="SearchResult-wrapper">
+      <Link
+        to={
+          addon
+            ? this.getAddonLink(addon, addonInstallSource)
+            : history.location.pathname
```

Comment:
I'm not sure this will give us what we want. It would redirect the user to the base page, but if there were any URL parameters they would be lost. I think what we want is a link that does nothing when clicked if there is no `addon`. Perhaps this could be achieved by using `href="#"` in the case where there is no `addon`?

Reply:
Also, as this is new logic, there should be a test added for the case where there is no `addon`, verifying that the `Link` is configured the way we expect it to be.

Reply:
Okay @bobsilverberg I will add new test case for this logic